sdk/src/builder.rs

```diff
@@ -112,6 +115,8 @@ fn default_vec<T>() -> Vec<T> {
     Vec::new()
 }
 
+/// Assertion data is almost always a JSON structure, but this allows for
```

Comment:
This comment applies to the Cbor variant of this enum. I think all we need to say here is that.
This allows the assertion to be expressed as CBOR or JSON. 
The default is CBOR unless you specify that an assertion should be JSON.

Reply:
Addressed.